index.js

```diff
@@ -55,6 +57,7 @@ function ensureAllCommentsPrinted(astComments) {
 }
 
 function formatWithCursor(text, opts, addAlignmentSize) {
+  text = prependFormatIfNecessary(text);
```

Comment:
> chance to ever strip the BOM character that is now not at the start of the file

do we have any such files which include a BOM? I would hope not as I'm pretty sure it causes a wealth of headaches in general. then again, I don't know what I'm talking about